lms/djangoapps/organizations/tests.py

```diff
@@ -308,6 +309,31 @@ def test_organizations_users_get(self):
         self.assertEqual(response.data[0]['username'], self.test_user.username)
         self.assertEqual(response.data[0]['email'], self.test_user.email)
 
+    def test_organizations_courses_get(self):
+        organization = self.setup_test_organization()
+        courses = CourseFactory.create_batch(2)
+        groups = GroupFactory.create_batch(2)
+
+        for i in xrange(2):
+            CourseGroupRelationship.objects.create(course_id=unicode(courses[i].id), group=groups[i])
+            groups[i].organizations.add(organization['id'])
+
+        test_uri = '{}{}/'.format(self.base_organizations_uri, organization['id'])
+        courses_uri = '{}courses/'.format(test_uri)
+        response = self.do_get(courses_uri)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data), 2)
+        self.assertEqual(response.data[0]['id'], unicode(courses[0].id))
+        self.assertEqual(response.data[1]['id'], unicode(courses[1].id))
+
+        # test organization course having multiple coursegrouprelationships
+        CourseGroupRelationship.objects.create(course_id=unicode(courses[0].id), group=groups[0])
```

Comment:
Should not the value of group field here be `groups[1]` to create first course relationship with second group `groups[1]`?.